page-optimize.php

```diff
@@ -21,6 +21,47 @@
 	exit;
 }
 
+if( ! wp_next_scheduled( 'page_optimize_cache_cleanup' ) ) {
+	wp_schedule_event( time(), 'daily', 'page_optimize_cache_cleanup' );
+}
+
+function page_optimize_cache_cleanup() {
+	if ( ! is_dir( PAGE_OPTIMIZE_CACHE_DIR ) ) {
+		return;
+	}
+
+	// Grab all files in the cache directory
+	$cache_files = array_filter(
+		scandir( PAGE_OPTIMIZE_CACHE_DIR ),
+		function ( $file ) {
+			return is_file( PAGE_OPTIMIZE_CACHE_DIR . "/$file" );
```

Comment:
Related to a comment below, we should probably have a better way to identify cache files specifically. Otherwise, we could get into trouble when the user sets PAGE_OPTIMIZE_CACHE_DIR to a directory with other potentially-writable files like `/srv/htdocs`.